python/lsst/ap/association/apDataProductMaker.py

```diff
@@ -0,0 +1,196 @@
+#
```

Comment:
Did you forget to rename the module? Typically we name the files something similar to the Task it contains so that it's easy to find. I wouldn't necessarily know that `apDataProductMaker.py` contains `APDataMapperTask`. We also usually name our tasks with the verb first: so `MapAPDataTask` and the module `mapApData.py`